install-lighthouse-reth.sh

```
@@ -0,0 +1,177 @@
+    sudo apt-get install --no-install-recommends --no-install-suggests -y $python-pip $python-tk $python-venv
+    ohai "Creating venv"
+    $python -m venv ~/.local --system-site-packages
+    ohai "Installing pip requirements"
+    ~/.local/bin/pip install requests console-menu python-dotenv tqdm
+    exit_on_error $?
+}
+
+linux_install_validator-install() {
+    ohai "Cloning ethpillar into ~/git/ethpillar"
+    mkdir -p ~/git/ethpillar
+    git clone https://github.com/coincashew/ethpillar.git ~/git/ethpillar 2> /dev/null || (cd ~/git/ethpillar ; git fetch origin main ; git checkout main ; git pull)
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)